src/components/ThreeCanvas.js

```diff
@@ -51,43 +51,6 @@ export const ThreeCanvas = ({ imageFile, locData, driverColor, driverSelected, f
     const control = new OrbitControls(camera, renderer.domElement);
     control.maxDistance = 10;
     control.minDistance = 5;
-    control.enableZoom = false;
```

Comment:
i get what you did here but what we had before was "better/closer to natural orbit control". explained in comment below how to get at least the track in the right position in the scene for the end result i'm picturing.